packages/p2p-v2/src/hooks/useAdvertiserStats.ts

```diff
@@ -21,11 +21,14 @@ const useAdvertiserStats = (advertiserId?: string) => {
             ...data,
 
             /** The average buy time in minutes */
-            averagePayTime: data?.buy_time_avg && data.buy_time_avg > 60 ? Math.round(data.buy_time_avg / 60) : 1,
+            averagePayTime: data?.buy_time_avg ? (data.buy_time_avg > 60 ? Math.round(data.buy_time_avg / 60) : 1) : -1,
 
             /** The average release time in minutes */
-            averageReleaseTime:
-                data?.release_time_avg && data.release_time_avg > 60 ? Math.round(data.release_time_avg / 60) : 1,
+            averageReleaseTime: data?.release_time_avg
+                ? data.release_time_avg > 60
+                    ? Math.round(data.release_time_avg / 60)
+                    : 1
+                : -1,
```

Comment:
@adrienne-deriv, is it possible for this to be extracted out into some kind of helper `function`, instead of nesting the `ternaries`?
 
```suggestion
            averageReleaseTime: data?.release_time_avg
                ? data.release_time_avg > 60
                    ? Math.round(data.release_time_avg / 60)
                    : 1
                : -1,
```